runtime/cpp/yolov5_yolov7_detection/yolov5_yolov7_inference.cpp

```diff
@@ -347,19 +349,38 @@ int main(int argc, char** argv) {
     }
     auto vdevice = vdevice_exp.release();
 
-    auto network_group_exp = configure_network_group(*vdevice, yolov_hef);
-    if (!network_group_exp) {
-        std::cerr << "Failed to configure network group " << yolov_hef << std::endl;
-        return network_group_exp.status();
+    auto network_group = configure_network_group(*vdevice, yolo_hef);
+    if (!network_group) {
+        std::cerr << "Failed to configure network group " << yolo_hef << std::endl;
+        return network_group.status();
     }
-    auto network_group = network_group_exp.release();
+    // auto network_group = network_group_exp.release();
 
-    auto vstreams_exp = VStreamsBuilder::create_vstreams(*network_group, QUANTIZED, FORMAT_TYPE);
-    if (!vstreams_exp) {
-        std::cerr << "Failed creating vstreams " << vstreams_exp.status() << std::endl;
-        return vstreams_exp.status();
+    auto input_vstreams_params = network_group.value()->make_input_vstream_params(QUANTIZED, FORMAT_TYPE_INPUT, HAILO_DEFAULT_VSTREAM_TIMEOUT_MS, HAILO_DEFAULT_VSTREAM_QUEUE_SIZE);
```

Comment:
why this api and not the one that was used before? this is longer. Used only if you want non default order or type.